aspnetcore/security/authentication/azure-ad-b2c-webapi.md

```diff
@@ -167,15 +167,17 @@ To make an authenticated request to the web API, a bearer token is required. Pos
+   |       <strong>Auth URL</strong>        | `https://login.microsoftonline.com/{tenant domain name}/oauth2/v2.0/authorize?p=B2C_1_SiUpIn` |  Replace *{tenant domain name}* with the tenant's domain name. **IMPORTANT**: This URL must have the same domain name as what is found in `AzureAdB2C.Instance` in the web API's *appsettings.json* file. See NOTE&dagger;.                                                  |
+   |       <strong>Client ID</strong>       |                *{enter the Postman app's <b>Application ID</b>}*                              |                                                                                                                                                                                                                                                                              |
+   |         <strong>Scope</strong>         |         `https://{tenant domain name}/{api}/user_impersonation openid offline_access`       | Replace *{tenant domain name}* with the tenant's domain name. Replace *{api}* with the App ID URI you gave the web API when you first registered it (in this case, `api`). The pattern for the URL is: `https://{tenant}.onmicrosoft.com/{api-id-uri}/{scope name}`.         |
+   |         <strong>State</strong>         |                                      *{leave blank}*                                          |                                                                                                                                                                                                                                                                              |
    | <strong>Client Authentication</strong> |                                Send client credentials in body                                |                                                                                                                                                                                                                                                                              |
 
+    > [!NOTE]
+    > &dagger;The **Auth URL** can be confusing. The policy settings dialog in the Azure Active Directory B2C portal displays two possible URLs: One in the format `https://login.microsoftonline.com/{tenant domain name}/{additional path information}`, and the other in the format `https://{tenant name}.b2clogin.com/{tenant domain name}/{additional path information}`. It's **critical** that the domain found in in `AzureAdB2C.Instance` in the web API's *appsettings.json* file matches the one used in the web app's *appsettings.json* file. This is the same domain used for the Auth URL field in Postman. Additionally, note that Visual Studio uses a slightly different URL format than what's displayed in the portal. As long as the domains match, the URL works.
```

Comment:
~Additionally~ pretty much banned in this repo.

Reply:
It's not really confusing - it's that there are two URI's. Consider removing ~The **Auth URL** can be confusing.~ and get straight to the problem:

It's **critical** that the domain found in in `AzureAdB2C.Instance` in the web API's *appsettings.json* file matches the one used in the web app's *appsettings.json* file. This is the same domain used for the Auth URL field in Postman. The policy settings dialog in the Azure Active Directory B2C portal displays two possible URLs: One in the format `https://login.microsoftonline.com/{tenant domain name}/{additional path information}`, and the other in the format `https://{tenant name}.b2clogin.com/{tenant domain name}/{additional path information}`. Visual Studio uses a slightly different URL format than what's displayed in the portal. As long as the domains match, the URL works.

Reply:
I'm not a fan of note chrome but in this case it's probably warranted. What's the difference between >Note and >Warning? Maybe warning is warranted?